### DISCUSSION TITLE
Unable To Use The Library

### DISCUSSION THREAD

**Discussion by CookedLobster** [charmbracelet/bubbletea#589] [Q&A] [answered]
I Manage to Import the Bubbletea Library, but I cannot run any of the Examples. I get the following error at the end of Every Program: 

p.Run undefined (type *tea.Program has no field or method Run)

**Comment by allyring** [comment, marked as answer]
`Run` has been replaced with `StartReturningModel`. If you replace the `Run` in the main function with `StartReturningModel`, it should start working.

**Reply by CookedLobster** [reply]
Thank You.

**Reply by muesli** [reply]
No, it's the other way around. `StartReturningModel` is the method that has been deprecated in favor of `Run`. You're simply looking at the docs of master while trying to build against the last stable release.
